changelog.d/9079.misc

```diff
@@ -0,0 +1 @@
+Fix failures when building Debian packages
```

Comment:
This probably belongs in the debian changelog, not here.

Reply:
Done